docs/HowToConnectExternally.md

```diff
@@ -0,0 +1,298 @@
+
+> **ACTION**: You can rule out any of the points above one by one.
+Start by checking the overall Pod status for the NuoDB domain and database.
+Some of the common troubleshooting steps are listed below, however, there might be additional verifications specific to your deployment.
+
+1. Verify that all AP, TE, and SM pods are reported _Ready_ using `kubectl get pods` command.
+2. Check the NuoDB domain and database using `nuocmd show domain` command.
+3. Verify the database availability inside the cluster is using the same connection properties as the application uses.
+To verify this, use the  `nuosql` tool inside an AP Pod.
+4. Make sure that `external-address` and/or `external-port` process labels are configured correctly on the TE database processes using `nuocmd --show-json-fields hostname,labels get processes` command.
+If you are using the `--enable-external-access` process option, verify that the configured values are the same as the `EXTERNAL-IP` shown for the Kubernetes service in `kubectl get services` output.
+If the value is not the same, restart the TE database process and verify again.
+5. Verify that the `LBQuery` or `LBPolicy` syntax is correct. If using `LBPolicy`, verify that the expected policies are configured in the NuoDB Admin using `nuocmd get load-balancers` and `nuocmd get load-balancer-config`.
+6. Check Kubernetes events for issues deploying cloud load balancers and external addresses.
+For more information on this, see the [TE does not start](#te-does-not-start) section.
+7. Verify that the cloud load balancer is provisioned and forwards traffic to the correct Kubernetes cluster.
+Check that its configuration is correct and modify the Kubernetes service annotations if needed.
+8. Verify that the configured security rules allow external access.
+
+### TE does not start
+
+TEs started with the `--enable-external-access` process option will wait for the `LoadBalancer` service IP address or hostname to be available before they start.
+In a case of a problem during NLB provisioning, the IP address will never be populated and the _engine_ container will fail.
+The following errors can be seen in the TE container logs:
+
+```text
+2021-12-15T07:43:05.015+0000 INFO  [admin-nuodb-cluster0-0:te-database-nuodb-cluster0-demo-55d664c8d7-bn57p] CustomAdminCommands Found service name=database-nuodb-cluster0-demo-balancer, type=LoadBalancer, selector={u'app': u'database-nuodb-cluster0-demo', u'component': u'te'}
+2021-12-15T07:43:05.015+0000 INFO  [admin-nuodb-cluster0-0:te-database-nuodb-cluster0-demo-55d664c8d7-bn57p] CustomAdminCommands Waiting for load balancer service database-nuodb-cluster0-demo-balancer ingress address...
+'start te' failed: Timeout after 120.0 sec waiting for ingress hostname in service database-nuodb-cluster0-demo-balancer
+```
+
+> **ACTION**: Verify that the `EXTERNAL-IP` for the service is available using `kubectl get services` command.
```

Comment:
If using the word command at the end, then I suggest adding "the" after using.  Or simply just remove the trailing word "command" as not needed.

Reply:
Will remove the trailing "command" across the document.